internal/cluster/distribution/eks/defaults.go

```diff
@@ -113,6 +113,15 @@ var defaultImageMap = []struct {
 			"us-west-2":      "ami-000a48e69e7695a4a", // US West (Oregon).
 		},
 	},
+	{
+		constraintForVersion("1.16"),
+		map[string]string{
+			// Kubernetes Version 1.16
+			"eu-north-1": "ami-030f9071e0d32b989", // EU (Stockholm).
+			"eu-west-3":  "ami-0170bd57fa0c260da", // EU (Paris).
+			"us-east-2":  "ami-0edc51bc2f03c9dc2", // US East (Ohio).
```

Comment:
@pbalogh-sa can help with the full list, but ideally in a separate PR.

Reply:
The plan was to have single pr

Reply:
The list is full now.